pandas/tests/frame/test_operators.py

```diff
@@ -340,95 +341,96 @@ def test_combineFrame(self):
 
         assert np.isnan(added['D']).all()
 
-        self_added = self.frame + self.frame
-        tm.assert_index_equal(self_added.index, self.frame.index)
+        self_added = float_frame + float_frame
+        tm.assert_index_equal(self_added.index, float_frame.index)
 
-        added_rev = frame_copy + self.frame
+        added_rev = frame_copy + float_frame
         assert np.isnan(added['D']).all()
         assert np.isnan(added_rev['D']).all()
 
         # corner cases
 
         # empty
-        plus_empty = self.frame + self.empty
+        plus_empty = float_frame + DataFrame({})
```

Comment:
So we have already a few fixtures for `empty_frame` (one in sparse.frame.conftest and another in the resample conftest). Maybe it makes sense to have one at the highest level conftest instead?

Reply:
we removed empty, this is ok (though should be ``DataFrame()``)